src/PerfView/PerfViewData.cs

```diff
@@ -5744,6 +5749,39 @@ protected internal override StackSource OpenStackSourceImpl(string streamName, T
                                     }
                                     sumCumMetric += cumMetric;
                                     cumCount++;
+
+                                    string GetAllocationType(CallStackIndex csi)
```

Comment:
Give this a method comment, if only so it is clear that this code is a nested function.   mention any local variables that it is mutating (in particular loadedModules and allocationTypeNames).